thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeExecutor.java

```diff
@@ -270,6 +278,29 @@ private void updateMergedAnomalyWeight(MergedAnomalyResultDTO anomalyMergedResul
         }
       }
 
+      // Transform Time Series
```

Comment:
this seems to be duplicated across detectiontaskrunner and mergeexecutor. Can we extract it to a method?